test/e2e/specs/max-length-test.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-unused-expressions */
+
+module.exports = {
+  tags: ['max-length'], // run this suite with 'yarn e2e --tag max-length'
+  beforeEach: function (browser) {
+    // Wait for form to be loaded
+    browser.url(browser.globals.devServerURL)
+    browser.url(browser.globals.devServerURL + '/max-length')
+  },
+  'Fields with too many characters should be invalid': function (browser) {
+    browser.options.desiredCapabilities.name = 'Fields with too many characters should be invalid'
+    browser.expect.element('#validate-btn').to.be.present
+    browser.click('#validate-btn')
+    browser.expect.element('#max-length-example').to.have.attribute('class').which.contains('vf-form-invalid')
+    browser.expect.element('#max-length-string-example').to.have.attribute('class').which.contains('vf-invalid')
+    browser.expect.element('#max-length-text-example').to.have.attribute('class').which.contains('vf-invalid')
+    browser.expect.element('#max-length-url-example').to.have.attribute('class').which.contains('vf-invalid')
+    browser.expect.element('#max-length-script-example').to.have.attribute('class').which.contains('vf-invalid')
+    browser.expect.element('#max-length-email-example').to.have.attribute('class').which.contains('vf-invalid')
+    browser.end()
+  },
+  'String with not too many values should be valid': function (browser) {
```

Comment:
Non grammatical sentence, maybe something like:  String with length shorter then max length should be valid